shuffle/cli/src/account.rs

```diff
@@ -204,3 +218,42 @@ fn encode_create_parent_vasp_account_script_function(
         ],
     ))
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use crate::shared;
+    use std::fs;
+    use tempfile::tempdir;
+
+    #[test]
+    fn test_check_node_deployed_if_localhost_used() {
```

Comment:
is this test is essentially just checking if the node directory exists vs checking if the node is actually deployed? A more robust check for `check_node_deployed_if_localhost_used` would be to see if node is running, or this is check done elsewhere?

Reply:
hmm you're right this check isn't done elsewhere... Thinking about how i can check for this. Maybe check if the port is used?